packages/salesforcedx-apex-debugger/src/constants.ts

```diff
@@ -6,6 +6,7 @@
  */
 
 export const DEFAULT_STREAMING_TIMEOUT = 14400;
+export const DEFAULT_REQUEST_TIMEOUT = 20000;
```

Comment:
Used in requestService.ts. Same default as IDE1.

Reply:
I remember that we made this configurable in IDE1. Do we need to lift this into its own settings. Some people were on very slow connection and they could set it up to 120000 ms.

Also, as a suggestion, for any time units, maybe we want to include the units so that we say `DEFAULT_REQUEST_TIMEOUT_MS`?